webextensions/background/background.js

```diff
@@ -250,7 +251,7 @@ export async function tryInitGroupTab(aTab) {
     return;
   browser.tabs.executeScript(aTab.apiTab.id, Object.assign({}, scriptOptions, {
     //file:  '/common/l10n.js'
-    file:  '/resources/l10n.js' // ES module does not supported as a content script...
+    file:  '/external/l10n-classic.js' // ES module does not supported as a content script...
```

Comment:
I use _classic_ as the term of traditional script type by [_classic script_](https://html.spec.whatwg.org/multipage/webappapis.html#classic-script) in whatwg html spec